atomic_reactor/plugins/pre_pull_base_image.py

```diff
@@ -303,14 +307,16 @@ def _get_manifest_list(self, image):
             return self.manifest_list_cache[image]
 
         manifest_list = get_manifest_list(image, image.registry,
-                                          insecure=self.parent_registry_insecure)
+                                          insecure=self.parent_registry_insecure,
+                                          dockercfg_path=self.parent_registry_dockercfg_path)
         if '@sha256:' in str(image) and not manifest_list:
             # we want to adjust the tag only for manifest list fetching
             image = image.copy()
 
             try:
                 config_blob = get_config_from_registry(image, image.registry, image.tag,
-                                                       insecure=self.parent_registry_insecure)
+                                                       insecure=self.parent_registry_insecure,
+                                                       dockercfg_path=self.parent_registry_dockercfg_path)
```

Comment:
One way around this would be to format it like:
```
config_blob = get_config_from_registry(
    image, ...
    ...
)
```
We've done that in other places.

Reply:
Fixed.